db/migrate/20241120211843_add_is_confidential_to_references.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsConfidentialToReferences < ActiveRecord::Migration[8.0]
+  def change
+    add_column :references, :is_confidential, :boolean
```

Comment:
Have we considered adding a default value? Probably `false`

Reply:
I did consider adding a default value, but I think it's best to make the users choice explicit. Setting a default of `false` would apply this value to all the existing references, which may not align with what we decide for that third journey.

We are still waiting to here back about the journey for existing references. If we set the default to `false` now, it could make things more complicated between references that were intentionally marked as false and those that  haven't been reviewed, since there would be no `nil` values to differentiate them.

Reply:
News hot off the press! 

We have just been informed that we want all previous references to be considered as confidential as that is effectively what they were. So in this case setting a default value of `true` makes sense. 

Also, If we set the default to `true` this negates the need for updating the ~500k reference tables in a data migration.